### PR TITLE
Bugs/recovery onboarding [#4498] [#4483]

### LINKED ISSUE #4498: Signing words are not shown when account is recovered
### Description

*Type*: Bug

*Summary*: Signing phrase onboarding missing when the account is recovered

#### Expected behavior
I want to know about 3 signing words when my account is recovered

#### Actual behavior
No wallet onboarding about signing phrase when the account is recovered

### Reproduction
- Open Status
- Recover account

### Additional Information
* Status version: nightly 28 may
* Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #4498, #4483

### CODE REVIEW

File: src/status_im/ui/screens/accounts/recover/events.cljs [hunk 1]
```diff
@@ -40,7 +40,8 @@
                      :signed-up?            true
                      :signing-phrase        phrase
                      :settings              (constants/default-account-settings)
-                     :wallet-set-up-passed? true}]
+                     :wallet-set-up-passed? false
+                     :seed-backed-up?       true}]
```
Comment (goranjovic) [inline src/status_im/ui/screens/accounts/recover/events.cljs:44]: Awesome

**Review by goranjovic — APPROVED** [review]

**Review by jeluard — APPROVED** [review]
2 line changed, 2 issues addressed!

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/Dgok3Q for branch PR-4500

<blockquote><div><strong><a href="https://i.diawi.com/Dgok3Q">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/qndYwA for branch PR-4500

<blockquote><div><strong><a href="https://i.diawi.com/qndYwA">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Review by mandrigin — APPROVED** [review]

**Pull request merged**